src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4852,4 +4858,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PeWritingFailure" xml:space="preserve">
     <value>An error occurred while writing the output file: {0}</value>
   </data>
+  <data name="ERR_SemiOrLBraceOrArrowExpected" xml:space="preserve">
+    <value>{ or ; or =&gt; expected</value>
```

Comment:
Well, if you insist I can do this. BUT, I browsed through all error messages and see unquoted tokens in many places:
```XML
  <data name="ERR_RbraceExpected" xml:space="preserve">
    <value>} expected</value>
  </data>
  <data name="ERR_LbraceExpected" xml:space="preserve">
    <value>{ expected</value>
  </data>
  <data name="ERR_BadNewExpr" xml:space="preserve">
    <value>A new expression requires (), [], or {} after type</value>
  </data>
  <data name="ERR_BadVarDecl" xml:space="preserve">
    <value>Expected ; or = (cannot specify constructor arguments in declaration)</value>
  </data>
  <data name="ERR_BadArraySyntax" xml:space="preserve">
    <value>Array type specifier, [], must appear before parameter name</value>
  </data>
  <data name="ERR_BadStackAllocExpr" xml:space="preserve">
    <value>A stackalloc expression requires [] after type</value>
  </data>
  <!-- etc. etc. etc. -->
```
Therefore I leave it for now and you should discuss internally whether you want to change the representation everywhere, or leave it.